src/datachain/lib/file.py

```diff
@@ -283,9 +282,8 @@ def get_uri(self):
     def get_path(self) -> str:
         """Returns file path."""
         path = unquote(self.get_uri())
-        fs = self.get_fs()
-        if isinstance(fs, LocalFileSystem):
-            # Drop file:// protocol
+        source = urlparse(self.source)
+        if source.scheme == "file":
```

Comment:
We always have `self.source` set, right?

Reply:
Normally yes - but if not, `get_fs()` would fail anyway.